README.md

```diff
@@ -2,7 +2,24 @@
 
 Refer this [Wiki Page](https://bahmni.atlassian.net/wiki/spaces/BAH/pages/299630726/Running+Bahmni+on+Docker) for Running Bahmni on Docker for detailed instructions.
 
-# To start Bahmni LITE or STANDARD using docker compose: 
-1. Go to bahmni-lite or bahmni-standard subfolder. For example: `cd bahmni-lite`.
-2. Execute script: `./run-bahmni.sh`. This will give you options for start/stop/view-logs/pull/reset/etc. 
-3. Ensure your `.env` file in the sub-folder has correct PROFILE configured, before executing the above commands.  
+## Running Bahmni LITE or STANDARD using docker compose: 
+1. Navigate to the relevant subfolder for your desired configuration. For example: `cd bahmni-lite`.
+2. Execute the script: `./run-bahmni.sh`. This script provides various options such as start, stop, view logs, pull updates, reset, etc.
+3. Before executing the above commands, ensure that your `.env` file in the sub-folder is correctly configured with the appropriate PROFILE.
+
+## Environment Variable Configuration For Bahmni Lite
+The `.env` and `.env.dev` files are used for configuring environment variables for the Bahmni Lite Docker setup. The `.env` file points to the `1.0.0` image tag, while `.env.dev` points to the `latest` image.
+
+By default `run-bahmni.sh` script runs with the `.env`, that uses the `1.0.0` images
+```shell
```

Comment:
If you want to run `docker compose` commands directly, then use the `--env-file` option to pass this file. e.g: `docker compose up --env-file .env.dev`